backend/app/assets/javascripts/spree/backend/templates/orders/shipment_tracking.hbs

```diff
@@ -0,0 +1,22 @@
+<th>
+  {{ human_attribute_name "spree/shipment" "tracking" }}
+</th>
+<td colspan="4">
+  {{#if editing}}
+    <input type="text" name="tracking" class="fullwidth" value="{{tracking}}">
+  {{else}}
+    {{#if tracking}}
+      {{ tracking }}
+    {{else}}
+      {{ t "no_tracking_present" }}
+    {{/if}}
+  {{/if}}
+</td>
+<td class="actions">
+  {{#if editing}}
+    <button class="js-save fa fa-check no-text with-tip" data-action="save" title="{{ t "actions.save" }}"></button>
```

Comment:
In order to use the same `submit` event we need to add 

```html
<button type="submit" form="tracking_form">
```

the `form` attribute is important here.